model/Licensing/Classes/ListedLicense.md

```diff
@@ -0,0 +1,30 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# ListedLicense
+
+## Summary
+
+A license that is listed on the SPDX License List.
+
+## Description
+
+A ListedLicense represents a License that is listed on the SPDX License List
+at https://spdx.org/licenses.
+
+## Metadata
+
+- name: ListedLicense
+- SubclassOf: License
+- Instantiability: Concrete
+
+## Properties
+
+- listVersionAdded
+  - type: xsd:string
+  - minCount: 0
+  - maxCount: 1
+- deprecatedVersion
```

Comment:
Maybe use a more consistend name, like:

```suggestion
- listVersionDeprecated
```

Reply:
The existing names tie to the current names in the license-list-XML schema:

https://github.com/spdx/license-list-XML/blob/e9849fde827b339e0e29988e17f167b4be4f7653/schema/ListedLicense.xsd#L52-L53